src/rust/kafka-metrics-exporter/tests/integration_tests.rs

```diff
@@ -21,13 +22,17 @@ mod integration_tests {
     };
     use tokio_stream::StreamExt;
 
+    fn brokers() -> String {
+        let bucket = std::env::var("KAFKA_ENDPOINT").expect("KAFKA_ENDPOINT");
```

Comment:
also this variable should probably be named to something like endpoint instead of bucket

Reply:
WOW good catch